openlibrary/plugins/openlibrary/stats.py

```diff
@@ -247,21 +239,25 @@ def page_unload_hook():
     graphite_stats.increment(web.ctx.graphiteRequestStats.to_metric())
 
 
-def increment_error_count(key):
+def increment_error_count(key: str) -> None:
     """
     :param str key: e.g. ol.exceptions or el.internal-errors-segmented
     """
     top_url_path = 'none'
     page_class = 'none'
     if web.ctx and hasattr(web.ctx, 'path') and web.ctx.path:
         top_url_path = _get_top_level_path_for_metric(web.ctx.path)
-        page_class = _get_path_page_name(web.ctx.path)
+        page_class = _get_path_page_name()
 
+    # Code that follows relies on these not being None, so alert ASAP if they are.
     exception_type, exception_value, tback = sys.exc_info()
+    assert exception_type is not None
+    assert exception_value is not None
```

Comment:
We do not want any falsey values?
```suggestion
    assert exception_type
    assert exception_value
```

Reply:
Good point! :)
I've fixed this.